packages/playground-ui/src/index.css

```
@@ -59,42 +64,42 @@
+  --surface3: oklch(0.968 0 0);
+  --surface4: oklch(0.929 0 0);
+  --surface5: oklch(0.929 0 0 / 0.9);
+  --surface6: oklch(0.869 0 0);
+
+  --accent1: oklch(0.627 0.194 149.214);
+  --accent2: oklch(0.577 0.245 27.325);
+  --accent3: oklch(0.546 0.245 262.881);
+  --accent5: oklch(0.623 0.214 259.815);
+  --accent6: oklch(0.666 0.179 58.318);
+
+  --accent1Dark: oklch(0.962 0.044 156.743);
```

Stylelint (17.9.0)

[error] Expected no empty line before custom property (custom-property-empty-line-before)

(custom-property-empty-line-before)